### DISCUSSION TITLE
Default theme, file not found?

### DISCUSSION THREAD

**Discussion by nednoodlehead** [helix-editor/helix#8171] [Q&A] [answered]
Hello, I installed Helix from source, which is located at `F:\Various Software\helix\helix`. Next, I opened up through powershell and opened the config file using `:config-open`, and I have the simple config of:

```toml
theme = "tokyonight_storm"

[editor.cursor-shape]
insert = "bar"
```

And when I write and run `:config-reload`, only the cursor is changed. It may also be important that the config file is at `AppData\Roaming\helix\config.toml` (maybe it is searching the local directory for that file).

I tried to delete it and make a config.toml inside of the source folder, but after reopening helix, and opening config, it made the config file back at the `AppData` path, and ignored the new file.

I only get the file not found error when i explcitly look for the theme using `:theme "tokyonight_storm"`

`Could not load theme: Theme: file not found for: tokyonight_storm`

I am also like, brand new to this editor (and this type of editor), so dumbing things down would help.

**Comment by the-mikedavis** [comment, marked as answer]
It sounds like the runtime directory can't be found - that's where the theme files are. Did you setup the runtime directory when building from source? https://docs.helix-editor.com/master/install.html#configuring-helixs-runtime-files

**Reply by nednoodlehead** [reply]
I did not set up the runtime directory. I just copied the runtime directory next to the `config.toml` file, so in `AppData/Roaming/helix`, and it now works as expected. It is also possible to see the status of it with `hx --health`, which I did not know..

**Comment by usmanr000** [comment]
In Pakistan #[8171](https://todayalerts.pk/) is used for BISP and Ehsaas Program
